packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.types.ts

```diff
@@ -122,6 +122,32 @@ export interface ICheckboxProps extends React.ButtonHTMLAttributes<HTMLElement |
    * Optional keytip for this checkbox
    */
   keytipProps?: IKeytipProps;
+
+  /**
+   * Optional controlled indeterminate visual state for checkbox. Use this if you want to control
+   * the indeterminate state. Setting indeterminate state takes visual precedence
+   * over checked or defaultChecked props given but does not affect checked state.
+   * This is should not be a toggleable state.
```

Comment:
`This is should…` 🤔

Reply:
I was wondering if it should be "should" since the user is passing their own removeIndeterminate callback handling how indeterminate is removed. As in, if they wanted to handle indeterminate differently than what is established in spec then they could. Or should we push them towards complying with spec?

Reply:
I would say if the user has the ability to control it multiple times during the life of the component it needs to say `should` as it sounds more as a recommendation.

Reply:
Sounds good, I'll push up the changes, thanks!